lib/push.js

```diff
@@ -78,7 +79,17 @@ var reasonToEvents = {
   }
 }
 
-module.exports = function (log, db) {
+module.exports = function (log, db, config) {
+  var vapid
+  if (config.vapidKeysFile) {
+    var vapidKeys = JSON.parse(fs.readFileSync(config.vapidKeysFile))
```

Comment:
Oh right, thanks, I had missed that.

FWIW this pattern has always seemed a bit odd to me, calling something that can execute code on a plain-old-data file.  But we control the file path so I guess there's no real danger in it.